src/hidapi/libusb/hid.c

```diff
@@ -42,7 +42,7 @@
 
 /* GNU / LibUSB */
 #include <libusb.h>
-#if !defined(SDL_PLATFORM_ANDROID) && !defined(NO_ICONV)
+#if !defined(__ANDROID__) && !defined(NO_ICONV)
```

Comment:
Check the history on this? I might have added these `#ifdefs`

Reply:
These added in 3b7b8f3c09491f4076b38baf170ad9f3aa249bad, so it's coming from upstream.